lib/pf/Authentication/Source/OpenIDSource.pm

```diff
@@ -33,6 +52,64 @@ Which module to use for DynamicRouting
 
 sub dynamic_routing_module { 'Authentication::OAuth::OpenID' }
 
+=head2 lookup_from_provider_info
+
+lookup_from_provider_info
+
+=cut
+
+sub lookup_from_provider_info {
+    my ($self, $pid, $info) = @_;
+    my $person_info = $self->map_to_person($info);
+    if ($person_info) {
+        person_modify($pid, %$person_info);
```

Comment:
I got following error when PF try to modify an existing user:
```
Caught exception in captiveportal::Controller::Root->dynamic_application "Undefined subroutine &pf::Authentication::Source::OpenIDSource::person_modify called at /usr/local/pf/lib/pf/Authentication/Source/OpenIDSource.pm line 65."
```

Reply:
Fixed the import